mne/io/eeglab/eeglab.py

```diff
@@ -106,8 +106,8 @@ def _get_info(eeg, montage, eog=()):
     return info
 
 
-def read_raw_eeglab(input_fname, montage=None, preload=False, eog=(),
-                    verbose=None):
+def read_raw_eeglab(input_fname, montage=None, eog=(), misc=(), event_id=None,
```

Comment:
you can't add arguments in the middle, can you? The `read_raw_eeglab` has already been released. I think the usual strategy is to add it before the verbose ...